packages/models-library/src/models_library/api_schemas_webserver/catalog.py

```diff
@@ -133,3 +135,52 @@ class Config(InputSchema.Config):
 class ServiceResourcesGet(api_schemas_catalog_services.ServiceResourcesGet):
     class Config(OutputSchema.Config):
         ...
+
+
+class DEVServiceGet(ServiceGet):
```

Comment:
While `/dev` is ok-ish. this looks a bit strange. I guess you intend it to be used from now onwards. I do not particularly like the `DEV`  prefix and would get rid of anything meaning "temporary" for "development".

Reply:
@GitHK we will remove those when we adapt to the front-end. They will then replace `catalog/services`

Reply:
That was my point, I would find a proper name form now, not in the future

Reply:
Talked in person. This model will be put in place of the original `ServiceGet`. It's find like this. My understanding was that they would go together in parallel for a while.